File: backend/src/main/java/com/ssafy/db/entity/Quest.java
package com.ssafy.db.entity;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

// 우리가 랜덤으로 뿌려주기
@Entity
@Table(name = "QUEST")
@Getter @Setter
public class Quest {
    @Id
    @GeneratedValue
    @Column(name = "Quest_ID")
    private Long quest_id;

    private String content;
    
    private int level;      // point를 주기 위해 작성함
}

File: mobile/app/src/main/java/com/team/teamrestructuring/dto/Quest.kt
package com.team.teamrestructuring.dto

data class Quest(
    var quest_id:Long,
    var content:String,
    var point:Int,
    var type:String?

){
    override fun toString(): String {
        return "Quest(quest_id=$quest_id, content='$content', point=$point, type=$type)"
    }
}
